learn/get-started/write-a-graphql-api-with-ballerina.md

```diff
@@ -14,49 +14,72 @@ redirect_from:
     - /learn/getting-started/writing-a-graphql-api-with-ballerina
 ---
 
-Due to the batteries included nature of the Ballerina language, there is no need to add any third-party libraries to implement the GraphQL API. The Ballerina standard library itself is adequate. In this guide, you will be writing a simple GraphQL service to serve a dummy dataset related to COVID-19.
+>**Info:** Due to the batteries included nature of the Ballerina language, there is no need to add any third-party libraries to implement the GraphQL API. The Ballerina standard library itself is adequate. 
 
 ## Set up the prerequisites
 
 To complete this tutorial, you need:
 
-1. A command terminal
+1. [Ballerina 2202.0.0 (Swan Lake) ](https://ballerina.io/learn/installing-ballerina/setting-up-ballerina/) or greater
 2. A text editor
     >**Tip:** Preferably, [Visual Studio Code](https://code.visualstudio.com/) with the [Ballerina extension](https://marketplace.visualstudio.com/items?itemName=WSO2.ballerina) installed.
-3. A [Ballerina installation](https://ballerina.io/learn/installing-ballerina/setting-up-ballerina/)
+3. A command terminal
 
-## Design the GraphQL endpoint
+## Understand the implementation
 
-Usually, a GraphQL endpoint is defined using a GraphQL schema. Some languages require the GraphQL schema to create a
-GraphQL service (schema-first approach) while others do not need the schema to create the service
-(code-first approach). Ballerina GraphQL package uses the latter. Therefore, you do not need the schema file to create
-your service, and instead, once you write the Ballerina service, the Ballerina GraphQL package will generate the schema.
+Usually, a GraphQL endpoint is defined using a GraphQL schema. Even though some languages require the GraphQL schema to create a GraphQL service (schema-first approach), Ballerina does not need the schema to create the service
+(code-first approach). Once you write the Ballerina service, the Ballerina GraphQL package will generate the schema.
 
-The GraphQL endpoint you are going to create in this guide will have two main operations, `Query` and `Mutation`. The
-`Query` type will be used to read the data in the data source, and the `Mutation` operation will be used to update the
-data in the data source.
+The GraphQL endpoint you will create in this guide will have two main operations, `query` and `mutation`. The `query` type will be used to read the data in the data source, and the `mutation` operation will be used to update the data in the data source.
 
-### The `Query` Type
-The `Query` type has two fields.
+### The `query` type
```

Comment:
The `Query` type is a pre-defined type name, it should start with a capital letter.

```suggestion
### The `Query` type
```